test/test.getch.ts

```diff
@@ -1112,22 +1112,6 @@ describe('🍂 defaults & instances', () => {
     assert.deepStrictEqual(res.data, {});
   });
 
-  it('should set content-type to application/json by default, for buffer', async () => {
```

Comment:
can we assert a test for what happens if a buffer does get sent in?

Reply:
Sure, it’s mostly a partial revert of:
- https://github.com/googleapis/gaxios/pull/374/files

but that can be added.